### PR TITLE
Bogavril/regionaltest

### LINKED ISSUE #4946: [Feature Request] Integration test with refrehs_in
### MSAL client type

Confidential

### Problem statement

We don't have any integration test around refresh_in. This is just to observe that ESTS issues it, to avoid regressions such as MSAL changes sku and ESTS stops sending it. Functioanal aspect of refresh_in is covered by other tests.

### Proposed solution

_No response_

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by bgavrilMS**
Fixes #4946

Just tests. And make some lab constants public, so we can write these tests in other libs.

### CODE REVIEW

**Review by gladjohn — APPROVED** [review]

**Comment by bgavrilMS** [comment]
@trwalke - here is an E2E test with ESTS-R; ESTS-R requires usage of an SN/I certificate and our lab app uses just that.

File: tests/Microsoft.Identity.Test.Integration.netcore/HeadlessTests/ClientCredentialsTests.NetFwk.cs [hunk 1]
```diff
@@ -50,9 +51,46 @@ public void TestInitialize()
...
+        // regression test based on SAL introducing a new SKU value and making ESTS not issue the refresh_in value
+        // This needs to run on .NET and .NET FWK to protect against MSAL SKU value changes
         [DataTestMethod]
-        [DataRow(Cloud.Public, TargetFrameworks.NetFx | TargetFrameworks.NetCore )]
-        [DataRow(Cloud.Adfs, TargetFrameworks.NetFx | TargetFrameworks.NetCore )]
+        [DataRow(true)]
+        [DataRow(false)]
+        public async Task RefreshOnIsEnabled(bool useRegional)
+        {
+            // if this test runs on AzureDevOps, disable it
```
Comment (trwalke) [inline tests/Microsoft.Identity.Test.Integration.netcore/HeadlessTests/ClientCredentialsTests.NetFwk.cs:61]: Shouldn't this say
```suggestion
            // if this test runs on local devbox, disable it
```

**Review by trwalke — APPROVED** [review]

**Pull request merged**